lms/djangoapps/certificates/signals.py

```diff
@@ -69,6 +70,7 @@ def listen_for_passing_grade(sender, user, course_id, **kwargs):  # pylint: disa
 
     If needed, generate a certificate task.
     """
+    emit_course_completed_tracking_event(user, course_id)
```

Comment:
A user can pass a course without completing it (without finishing or even attempting all of its content).

If you're trying to emit an event when a user passes a course, the event should probably be renamed and emitted by the course grade code: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades